src/test/java/Homework15.java

```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework15 extends BaseTest {
+
+    @Test
+    public void search () {
+        String songName = "Pluto";
+
+        login();
+
+        WebElement searchForm = driver.findElement(By.cssSelector("#searchForm>input"));
+        searchForm.sendKeys(songName);
+        WebElement match = driver.findElement(By.cssSelector(".songs .details"));
+        String matchText = match.getText();
+        Assert.assertTrue(matchText.contains(songName));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/200160327-bc7f58c7-d14d-450e-85e5-32c7640bc4e7.png)

I fixed the code, try this:

    ```
    @Test
    public void search () {
        String songName = "Pluto";
        login();
        WebElement searchForm = driver.findElement(By.cssSelector("#searchForm>input"));
        searchForm.click();
        searchForm.sendKeys(songName);
        WebElement match = driver.findElement(By.cssSelector(".songs .details"));
        String matchText = match.getText();
        Assert.assertTrue(matchText.contains(songName));
    }
    ```
And also added in the class BaseTest:

          ```
         ChromeOptions chromeOptions = new ChromeOptions();
        chromeOptions.addArguments("--disable-notifications");
        driver = new ChromeDriver(chromeOptions);
         ```
         
![image](https://user-images.githubusercontent.com/104761482/200161975-bd0f563c-645c-4e2a-8d57-b77822d7fc22.png)

      
      and

    `driver.manage().window().maximize();`
![image](https://user-images.githubusercontent.com/104761482/200161987-ab3d2759-1bb0-42a8-8727-12055669229b.png)